MaxText/layers/linears.py

```diff
@@ -458,7 +459,9 @@ def generate_masks(self, top_k_indices, softmax_probs):
     # calculate expert_capacity = (tokens_per_batch / num_experts) * capacity_factor
     batch_size, seq_len, _ = top_k_indices.shape
     tokens_per_batch = seq_len * self.num_experts_per_tok
-    expert_capacity_per_batch = int((tokens_per_batch / self.num_experts) * self.config.capacity_factor)
+    expert_capacity_per_batch = int(
```

Comment:
Could you add a comment for future readers?

Reply:
Is there an extra int? `int(int(....))`?

Reply:
Yes, there was an extra int, removed it.